website/content/en/pre-docs/concepts/_index.md

```diff
@@ -0,0 +1,167 @@
+---
+title: "Concepts"
+linkTitle: "Concepts"
+weight: 35
+---
+
+Users fall under two basic roles: Kubernetes cluster operators and application developers.
+This document describes Karpenter concepts through the lens of those two types of users.
+
+## Cluster operator
+
+As a Kubernetes cluster operator, you can engage with Karpenter to:
+
+* Install Karpenter 
+* Configure provisioners to set constraints and other features for managing nodes
+* Deprovision nodes
+* Upgrade nodes
+
+Concepts associated with this role are described below.
+
+
+### Installing Karpenter
+
+Karpenter is designed to run on a node in your Kubernetes cluster.
+As part of the installation process, you need credentials from the underlying cloud provider to allow nodes to be started up and added to the cluster as they are needed.
+
+[Getting Started with Karpenter on AWS](https://karpenter.sh/docs/getting-started/)
+describes the process of installing Karpenter on an AWS cloud provider.
+Because requests to add and delete nodes and schedule pods are made through Kubernetes, AWS IAM Roles for Service Accounts (IRSA) are needed by your Kubernetes cluster to make privileged requests to AWS.
+For example, Karpenter uses AWS IRSA roles to grant the permissions needed to describe EC2 instance types and create EC2 instances.
```

Comment:
This is a bit confusing because Karpenter doesn't use IRSA. Karpenter uses a Kubernetes service account mounted in the pod. How that service account get's access into AWS depends on the implementation.